implementations/rust/ockam/ockam_command/src/value_parsers.rs

```diff
@@ -38,23 +38,28 @@ pub fn parse_enrollment_ticket(value: &str) -> miette::Result<EnrollmentTicket>
     }
 }
 
-fn parse_string_or_path_or_url(value: &str) -> miette::Result<String> {
-    parse_path_or_url(value).or_else(|_| Ok(value.to_string()))
+async fn parse_string_or_path_or_url(value: &str) -> miette::Result<String> {
+    parse_path_or_url(value)
+        .await
+        .or_else(|_| Ok(value.to_string()))
 }
 
-pub fn parse_path_or_url(value: &str) -> miette::Result<String> {
+pub async fn parse_path_or_url(value: &str) -> miette::Result<String> {
```

Comment:
What's the rationale behind making this async? This parser was used by clap [before](https://github.com/build-trust/ockam/blob/37adf739e6b640e3614b05d9b29fc1cb14025fc5/implementations/rust/ockam/ockam_command/src/entry_point.rs#L33) any async runtime is started.